<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
 <author>
    SPDX-FileCopyrightText: none
    SPDX-License-Identifier: GPL-3.0-or-later
  </author>
 <class>WdgBristleOptions</class>
 <widget class="QWidget" name="WdgBristleOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>357</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>425</width>
    <height>285</height>
   </size>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item colspan="2" column="0" row="9">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>162</height>
      </size>
     </property>
    </spacer>
   </item>
   <item colspan="2" column="0" row="0">
    <layout class="QGridLayout" columnstretch="0,1" name="gridLayout">
     <item column="0" row="1">
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Random offset:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item column="1" row="3">
      <widget class="KisDoubleSliderSpinBox" name="densityBox"/>
     </item>
     <item column="1" row="2">
      <widget class="KisDoubleSliderSpinBox" name="shearBox"/>
     </item>
     <item column="1" row="0">
      <widget class="KisDoubleSliderSpinBox" name="scaleBox"/>
     </item>
     <item column="1" row="7">
      <widget class="QCheckBox" name="antialiasCBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item column="1" row="4">
      <widget class="QCheckBox" name="mousePressureCBox">
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item column="1" row="6">
      <widget class="QCheckBox" name="connectedCBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item column="1" row="1">
      <widget class="KisDoubleSliderSpinBox" name="rndBox"/>
     </item>
     <item column="0" row="2">
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Shear:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item column="0" row="0">
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Scale:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item column="0" row="3">
      <widget class="QLabel" name="density">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Density:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item column="1" row="5">
      <widget class="QCheckBox" name="thresholdCBox">
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item column="1" row="8">
      <widget class="QCheckBox" name="compositingCBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item column="0" row="4">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Mouse pressure:</string>
       </property>
      </widget>
     </item>
     <item column="0" row="5">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Threshold:</string>
       </property>
      </widget>
     </item>
     <item column="0" row="6">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Connect hairs:</string>
       </property>
      </widget>
     </item>
     <item column="0" row="7">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Anti-aliasing:</string>
       </property>
      </widget>
     </item>
     <item column="0" row="8">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Composite bristles:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisDoubleSliderSpinBox</class>
   <extends>QWidget</extends>
   <header>kis_slider_spin_box.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>